projects/uitdatabank/docs/errors.md

```diff
@@ -77,3 +77,13 @@ Example error response:
 *   **Status**: `400`
 
 The `workflowStatus` of an event can only transition from `DRAFT` to `READY_FOR_VALIDATION`, from `READY_FOR_VALIDATION` to either `APPROVED` or `REJECTED`, or from any workflowStatus to `DELETED`. Other transitions are not possible, for example back from `REJECTED` to `READY_FOR_VALIDATION`.
+
+## attendance-mode-not-supported
+
+*   **Complete type:** `https://api.publiq.be/probs/uitdatabank/attendance-mode-not-supported`
+*   **Title**: `Attendance mode not supported`
+*   **Status**: `400`
+
+The action you are trying to perform on an event cannot be done because its `attendanceMode` does not support this action.
+
+For example, you cannot update the location of an event with attendanceMode `online` to a physical location. Or you cannot set the location of an event with attendanceMode `mixed` or `offline` to the nil location (`https://io.uitdatabank.be/places/00000000-0000-0000-0000-000000000000`).
```

Comment:
Everything was changed to `nil location` in the code, but inside documentation, I still used `online location` because that is clearer for the current context.

Dot we also use `nil location` in documentation?

Reply:
To me in a technical context `nil location` is clearer personally because nil = 0 and `00000000-0000-0000-0000-000000000000` is officially called the nil uuid. But maybe I'm biased because I happen to know that term for the uuid while the average integrator might not.

On the other hand I'm not sure that `00000000-0000-0000-0000-000000000000` = `online location` would be clear at first glance without reading about online events and attendance mode first. 🤔

Reply:
Ok, I will make sure to use `nil location` inside the code and inside documentation.

Reply:
As discussed on Slack we'll keep calling it nil location in the docs so this does not need to be changed. But we'll probably need to change some other occurences of "online location" in the docs to "nil location". And also clarify what it means in the terminology page.